### DISCUSSION TITLE
I generated my static website with Dioxus.

### DISCUSSION THREAD

**Discussion by tiswas247** [DioxusLabs/dioxus#2804] [General]
I normally use zola but decided I didn't like writing templates and wanted to use components.

The code is here https://github.com/bionic-gpt/bionic-gpt/tree/main/crates/static-website and the finished website is here https://bionic-gpt.com/

It's hosted on cloudflare pages.

**Comment by ThomasCartier** [comment]
Well done, careful with the i18n  translation though:
![Screenshot_20240819_145250](https://github.com/user-attachments/assets/9e182590-cae3-41b4-a30e-acd7b78e9382)

See the `{text1}` and `{text2}`?
They are throughout the page..

Cheers,

**Reply by tiswas247** [reply]
Thanks.
